baselines/webworker.generated.d.ts

```diff
@@ -5876,9 +5888,9 @@ type NamedCurve = string;
 type OnErrorEventHandler = OnErrorEventHandlerNonNull | null;
 type PerformanceEntryList = PerformanceEntry[];
 type PushMessageDataInit = BufferSource | string;
-type ReadableStreamController<T> = ReadableStreamDefaultController<T>;
-type ReadableStreamReadResult<T> = ReadableStreamReadValueResult<T> | ReadableStreamReadDoneResult;
-type ReadableStreamReader<T> = ReadableStreamDefaultReader<T>;
+type ReadableStreamController<T> = ReadableStreamDefaultController<T> | ReadableByteStreamController;
```

Comment:
This could break existing code, for example:
```javascript
new ReadableStream({
  start(controller) {
    controller.enqueue("a");
  }
});
```
We don't yet recognize that this is a non-byte stream, so we leave `controller` as `ReadableStreamDefaultController<T> | ReadableByteStreamController`. But that means you can **only** pass byte chunks to `enqueue()`.

I could investigate if we can change the `ReadableStream` constructor to have separate overloads for constructing byte streams and non-byte streams?

Reply:
I gave it a go! 😁